cmd/serverless-init/main.go

```diff
@@ -35,15 +38,32 @@ const (
 	loggerName        = "SERVERLESS_INIT"
 )
 
+type cliParams struct {
```

Comment:
Do we really need to define this type? Or is it a golang best practice?

Reply:
It's a convention we're following for One-Shot Fx apps. Here's the internal documentation for it: https://github.com/DataDog/datadog-agent/blob/921e96c33fb884720a6487b1c41d6ba373746adc/docs/components/defining-apps.md#one-shot-apps